web_widget_dropdown_dynamic/README.rst

```diff
@@ -89,12 +89,14 @@ Credits
 Authors
 ~~~~~~~
 
-* Brainbean Apps OU
+* CorporateHub
 
 Contributors
 ~~~~~~~~~~~~
 
-* Alexey Pelykh <alexey.pelykh@brainbeanapps.com>
+* `CorporateHub <https://corporatehub.eu/>`__
```

Comment:
you should change it in readme/CONTRIBUTORS.rst ;)

Reply:
@simahawk yeah, I know, but in any case README will be regenerated

Reply:
eh, you changed something that will be overridden w/ the old value :wink:

Reply:
https://github.com/OCA/web/blob/2069abd6b78f5b56212d8ad888a3ee896f3c66e1/web_widget_dropdown_dynamic/readme/CONTRIBUTORS.rst

Reply:
Oh, that's a good catch! I wonder how I did miss that, thanks

Reply:
@simahawk fixed, thanks!